assets/stylesheets/modules/speakers.scss

```diff
@@ -145,6 +149,37 @@
   }
 }
 
+.l-speaker__bio {
+  margin-bottom: 12px;
+  position: relative;
+}
+
+.speaker__bio__info {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba($pink, .8);
+  border-radius: 50%;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  border: 0px solid $pink;
+  opacity: 0;
+  @include transition-duration(.25s);
+  @include transition-property(border,opacity);
```

Comment:
http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/
scale is a higher performing animation. instead can we always have the border there and scale it in?